src/components/split-button/split-button-test.stories.tsx

```diff
@@ -53,44 +54,39 @@ export default {
   },
 };
 
-/* eslint-disable react/prop-types */
+type SplitButtonStoryArgs = {
+  buttonType?: SplitButtonProps["buttonType"];
```

Comment:
```suggestion
  buttonType: SplitButtonProps["buttonType"];
```

I don't think you need a `?` here as `buttonType` is already optional in `SplitButtonProps`.

Reply:
Removed.